crates/matrix-sdk/src/room_preview.rs

```diff
@@ -219,6 +250,105 @@ impl RoomPreview {
 
         let state = client.get_room(room_id).map(|room| room.state());
 
-        Ok(Self::from_room_info(room_info, num_joined_members, state))
+        Ok(Self::from_room_info(client, room_info, num_joined_members, state).await)
     }
 }
+
+pub enum RoomPreviewActions {
```

Comment:
In this entire file you need to comment each struct, enum, variant, field, function. I'll enable a new lint for this :smiling_imp:

Reply:
This was fixed in the following commit if I'm not mistaken.